docs/utilities/tags.md

```diff
@@ -33,24 +33,24 @@ You may declare a `remove` array of the same format as the `values` array. Any v
 
 Using Tags In Code
 ------------------
-Block, Item, Fluid, and EntityType tags are automatically sent from the server to any remote clients on login and reload. In addition, Forge adds BlockEntityType, Potion, and Enchantment tags. Function tags are not synced.
+Tags for all registries are automatically sent from the server to any remote clients on login and reload. However, only tags for `Block`s, `Item`s, `EntityType`s, `Fluid`s, and `GameEvent`s can be held by their registry objects.
```

Comment:
I believe this is slightly inaccurate; it is not that only these five objects hold their tags, but rather that the tags in which they are contained in are accessible through the `Holder` available through `#builtInRegistryHolder()`. However, we may not want to mention this method, given that these are marked deprecated, presumably because Mojang plans to eliminate them in a future version.